priv/repo/data/channels.json

```
@@ -198,5 +198,15 @@
     "name": "Ed Codes - Shopify Tutorials",
     "image_url": "https://yt3.ggpht.com/KMgKgdIy1kc_13HDJXvKuN5fewpysTvqdXZMy_c8LL5LzlSDhhiCguqn9jWBx-_3My6jY9p_bgQ=s88-c-k-c0x00ffffff-no-rj",
     "youtube_channel_id": "UCfk-c_Vx8Wo5Zu5dH96Yw0w"
+  },
+  {
+    "name": "Kunal Kushwaha",
+    "image_url": "https://yt3.googleusercontent.com/ytc/AIdro_mz7KssXk5zh8YWHnef2va6D5s0tK3Q4f59RNHajTM8tg=s160-c-k-c0x00ffffff-no-rj",
+    "youtube_channel_id": "UCxD5EE0H7qOhRr0tIVsOZPQ"
+  },
+  {
+    "name": "Aprenda Go",
+    "image_url": "https://yt3.googleusercontent.com/qCuUkim8fjMmWqE9I55AH1lC5QQR8NcsPXR29DnKvXctB8OUgTYxaszYmo-x-e0xF3rW5gRo=s160-c-k-c0x00ffffff-no-rj",
+    "youtube_channel_id": "UCBGOUQHNNtNGcGzVq5rIXjw"
```

I was checking the channels on YouTube and these 2 seems to be inverted

```suggestion
  {
    "name": "Aprenda Go",
    "image_url": "https://yt3.googleusercontent.com/ytc/AIdro_mz7KssXk5zh8YWHnef2va6D5s0tK3Q4f59RNHajTM8tg=s160-c-k-c0x00ffffff-no-rj",
    "youtube_channel_id": "UCxD5EE0H7qOhRr0tIVsOZPQ"
  },
  {
    "name": "Kunal Kushwaha",
    "image_url": "https://yt3.googleusercontent.com/qCuUkim8fjMmWqE9I55AH1lC5QQR8NcsPXR29DnKvXctB8OUgTYxaszYmo-x-e0xF3rW5gRo=s160-c-k-c0x00ffffff-no-rj",
    "youtube_channel_id": "UCBGOUQHNNtNGcGzVq5rIXjw"
```